src/store/configureStore.dev.js

```diff
@@ -3,13 +3,17 @@
 // With Redux, the actual stores are in /reducers.
 
 import {createStore, compose, applyMiddleware} from 'redux';
+import reduxImmutableStaeInvariant from 'redux-immutable-state-invariant';
```

Comment:
reduxImmutableStaeInvariant is missing a "t". Should be reduxImmutableStateInvariant.

:)